boards/arm/nrf52840_mdk/nrf52840_mdk.dts

```diff
@@ -97,8 +97,6 @@
 
 &pwm0 {
 	status = "okay";
-	ch0-pin = <22>;
-	ch0-inverted;
```

Comment:
By removing the only pin assigned to this node you make it actually unusable (no `pwm_pin_set_*` function can be successfully executed for the associated device, see [pwm_nrfx.c](https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/pwm/pwm_nrfx.c#L141)), so the status of the node can no longer be "okay".

But I think that the preparation of these boards for #20657 should be rather done by adding of "pwm-leds" compatible nodes for them, like it was done in [this commit](https://github.com/zephyrproject-rtos/zephyr/commit/32e194be1631a4ae044437f3c8d18e0bdf031a98#diff-0ad92537b7c3e2041e8e2a6b9a30698b) for Nordic DKs.

Reply:
@anangl Thank you for your feedback. I have changed the patches to add PWM LEDs instead. I have only compile-tested these changes as I do not have access to any of these boards.

Reply:
> I have only compile-tested these changes as I do not have access to any of these boards.

Neither do I. I can only comment the changes basing on the knowledge of the Nordic PWM peripheral and the Zephyr driver for it.